coss/discourse/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+from wagtail.wagtailsnippets.models import register_snippet
+
+@register_snippet
+class DiscourseCategoryLatest(models.Model):
+    category_id = models.PositiveSmallIntegerField()
```

Comment:
Oh I see - this is to store the ID of the Discourse category which should be queried by the snippet. Doesn't necessarily have to be stored as an integer, but that's how it's done in Discourse.